web/themes/custom/my_awesome_theme/components/03-organisms/site/site-gadgets-page/_site-gadgets-page.scss

```diff
@@ -1,5 +1,6 @@
-/**.layout--onecol{
+
 @todo use this code for fashion taxonomy page
```

Comment:
Is that scss even compiles
You removed start of the comment, left the end, and also this @todo seems outdated now.